common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml

```diff
@@ -202,10 +202,12 @@ eActionOp_APMANAGER:
   _type: enum
   _enum_storage: uint8_t
   ACTION_APMANAGER_INIT_DONE_NOTIFICATION: 0
-  ACTION_APMANAGER_JOINED_NOTIFICATION: 1
-  ACTION_APMANAGER_ENABLE_APS_REQUEST: 2
-  ACTION_APMANAGER_ENABLE_APS_RESPONSE: 3
-  ACTION_APMANAGER_ACK: 4
+  ACTION_APMANAGER_UP_NOTIFICATION: 1
```

Comment:
```suggestion
  ACTION_APMANAGER_IFACE_UP_NOTIFICATION: 1
```

Reply:
This message doesn't notify about an interface that went up, it notifies about the ap_manager thread is up and running.